docs/src/main/docs/guides/07_health_mp_guide.adoc

```diff
@@ -0,0 +1,542 @@
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Health Check MP Guide
+:description: Helidon health-checks
+:keywords: helidon, health-checks, health, check
+
+This guide describes how to create a sample MicroProfile (MP) project
+that can be used to run some basic examples using both built-in and custom health-checks with Helidon MP.
+
+== What you need
+
+[width=50%,role="flex, sm7"]
+|===
+|About 15 minutes
+|<<about/03_prerequisites.adoc,Helidon Prerequisites>>
+|===
+
+=== Create a sample MP project
+
+Generate the project sources using the Helidon MP Maven archetype.
+The result is a simple project that can be used for the examples in this guide.
+
+[source,bash,subs="attributes+"]
+.Run the Maven archetype:
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-mp \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-mp \
+    -Dpackage=io.helidon.examples.quickstart.mp
+----
+
+=== Using the built-in health-checks
+
+Helidon has a set of built-in health-checks that are automatically enabled to report various
+health-check statuses that are commonly used:
+
+* deadlock detection
+* available disk space
+* available heap memory
+
+The following example will demonstrate how to use the built-in health-checks.  These examples are all executed
+from the root directory of your project (helidon-quickstart-mp).
+
+[source,bash]
+.Build the application, skipping unit tests, then run it:
+----
+mvn package -DskipTests=true
+java -jar target/helidon-quickstart-mp.jar
+----
+
+[source,bash]
+.Verify the health endpoint in a new terminal window:
+----
+curl http://localhost:8080/health
+----
+
+[source,json]
+.JSON response:
+----
+{
+  "outcome": "UP",
+  "status": "UP",
+  "checks": [
+    {
+      "name": "deadlock",
+      "state": "UP",
+      "status": "UP"
+    },
+    {
+      "name": "diskSpace",
+      "state": "UP",
+      "status": "UP",
+      "data": {
+        "free": "325.54 GB",
+        "freeBytes": 349543358464,
+        "percentFree": "69.91%",
+        "total": "465.63 GB",
+        "totalBytes": 499963174912
+      }
+    },
+    {
+      "name": "heapMemory",
+      "state": "UP",
+      "status": "UP",
+      "data": {
+        "free": "230.87 MB",
+        "freeBytes": 242085696,
+        "max": "3.56 GB",
+        "maxBytes": 3817865216,
+        "percentFree": "98.90%",
+        "total": "271.00 MB",
+        "totalBytes": 284164096
+      }
+    }
+  ]
+}
+----
+
+NOTE: In MicroProfile Health 2.0 outcome and state were replaced by status in the JSON response wire format.
+Helidon currently provides both fields for backwards compatibility, but use of outcome and state is deprecated
+and will be removed in a future release. You should rely on state instead.
```

Comment:
"you should rely on **status** instead."
Also, might want to back tick outcome, state and status in this paragraph so they stand out as "code".

Reply:
fixed